polkadot/node/network/statement-distribution/src/v2/cluster.rs

```diff
@@ -424,6 +425,25 @@ impl ClusterTracker {
 	fn is_in_group(&self, validator: ValidatorIndex) -> bool {
 		self.validators.contains(&validator)
 	}
+
+	/// Dumps pending statement for this cluster.
+	///
+	/// Normally we should not have any pending statements for our cluster,
+	/// but if we do for long periods of time something bad happened which
+	/// needs to be investigated.
+	pub fn dump_pending_statements(&self, parent_hash: Hash) {
+		if self.pending.iter().filter(|pending| !pending.1.is_empty()).count() >=
+			self.validators.len()
```

Comment:
just to clarify, the commend above says the left hand side should normally be zero, then why do we use `self.validators.len()` as a threshold for the warning? is it an arbitrary threshold?

Reply:
Fix the comment, it is actually a problem when we have pending statements to all validators in our backing group.